test/Traveler-test.js

```diff
@@ -59,7 +59,14 @@ describe('Traveler', () => {
     ]);
   });
 
-  it.only('should be able to get pending trips', () => {
+  it('should be able to retrieve their destination Ids', () => {
+    agency = new Agency(travelersTestData, tripTestData, destinationTestData);
```

Comment:
Could these agency instantiations be done inside of the before each to save you some lines?

Reply:
You are right, at first i didnt need them, but when I changed the class structure, I kept needing them. Thanks for catching that